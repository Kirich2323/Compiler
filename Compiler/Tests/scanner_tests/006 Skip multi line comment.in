{ Multiline comment
Second line
  Third Line
}
a