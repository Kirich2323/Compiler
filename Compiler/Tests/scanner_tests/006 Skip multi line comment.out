5    1   a               a               Identifier
